assets/sass/_fonts.scss

```diff
@@ -0,0 +1,29 @@
+// Should be preloaded in for-head.mustache
+@font-face {
+  font-display: fallback;
+  font-family: "Noto Sans";
+  src: url("../fonts/NotoSans-Regular-webfont-custom-2-subsetting.woff2") format("woff2");
```

Comment:
@thewilkybarkid Will these paths (this one and the three more below on lines 12, 20 & 27) be okay for cache busting? I think we need to retain the information that exists in the filename, but can reformat it if that would make things easier.

Reply:
Should be fine (we do the same for images already).